src/script/page/RightSidebar/ConversationDetails/ConversationDetailsOptions.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {RECEIPT_MODE} from '@wireapp/api-client/src/conversation/data/';
+import {FC} from 'react';
+
+import Icon from 'Components/Icon';
+import ReceiptModeToggle from 'Components/toggle/ReceiptModeToggle';
+
+import {t} from 'Util/LocalizerUtil';
+
+import {PanelParams, PanelViewModel} from '../../../view_model/PanelViewModel';
+import {Conversation} from '../../../entity/Conversation';
+
+interface ConversationDetailsOptionsProps {
+  activeConversation: Conversation;
+  togglePanel: (state: string, params: PanelParams) => void;
+  receiptMode: RECEIPT_MODE;
+  guestOptionsText: string;
+  notificationStatusText: string;
+  servicesOptionsText: string;
+  timedMessagesText: string;
+  showOptionGuests: boolean;
+  showOptionNotificationsGroup: boolean;
+  showOptionReadReceipts: boolean;
+  showOptionServices: boolean;
+  showOptionTimedMessages: boolean;
+  updateConversationReceiptMode: (receiptMode: RECEIPT_MODE) => void;
+}
+
+const ConversationDetailsOptions: FC<ConversationDetailsOptionsProps> = ({
+  activeConversation,
+  togglePanel,
+  receiptMode,
+  guestOptionsText,
+  notificationStatusText,
+  servicesOptionsText,
+  showOptionGuests,
+  showOptionNotificationsGroup,
+  showOptionReadReceipts,
+  showOptionServices,
+  showOptionTimedMessages,
+  timedMessagesText,
+  updateConversationReceiptMode,
+}) => {
+  const onNotificationsClick = () => togglePanel(PanelViewModel.STATE.NOTIFICATIONS, {entity: activeConversation});
+
+  const onTimedMessagesClick = () => togglePanel(PanelViewModel.STATE.TIMED_MESSAGES, {entity: activeConversation});
+
+  const onGuestOptionsClick = () => togglePanel(PanelViewModel.STATE.GUEST_OPTIONS, {entity: activeConversation});
+
+  const onServiceOptionsClick = () => togglePanel(PanelViewModel.STATE.SERVICES_OPTIONS, {entity: activeConversation});
```

Comment:
Let's name those with the action they are doing, not the time at which they are triggered:

```suggestion
  const openNotificationsPanel = () => togglePanel(PanelViewModel.STATE.NOTIFICATIONS, {entity: activeConversation});

  const openTimedMessagePanel = () => togglePanel(PanelViewModel.STATE.TIMED_MESSAGES, {entity: activeConversation});

  const openOptionsPanel = () => togglePanel(PanelViewModel.STATE.GUEST_OPTIONS, {entity: activeConversation});

  const openServicePanel = () => togglePanel(PanelViewModel.STATE.SERVICES_OPTIONS, {entity: activeConversation});
```